packages/core/content-releases/server/src/types.ts

```diff
@@ -0,0 +1,33 @@
+// @TODO: Probably user & role types should be imported from a common package
+interface RoleInfo {
+  id: number | string;
+  name: string;
+  code: string;
+  description?: string;
+  usersCount?: number;
+}
+
+export interface UserInfo {
```

Comment:
What do you think to use the one in the helper plugin https://github.com/strapi/strapi/blob/1c1e34d14f7d01aca6f6e0194858bff049caa1c3/packages/core/helper-plugin/src/utils/auth.ts#L13 ?

Reply:
or extend it

Reply:
I didn't use it because we would need to change the helper plugin (to export this type) and in other sides we are not using it, but definitely in the future we need to export this interface from a shared place, probably from strapi package 🤔